### PR TITLE
geanyctags, projectorganizer, vimode: Some README and URL updates

### PR DESCRIPTION
**Pull request by techee**
1. Suggest using Github for reporting bugs and pull requests instead of sending them by email.
2. Use https:// instead of http://
3. Mention universal-ctags as a maintained version of ctags in geanyctags README

### CODE REVIEW

File: geanyctags/README [hunk 1]
```diff
@@ -11,7 +11,9 @@ GeanyCtags adds a simple support for generating and querying ctags files for a G
 project. It requires that the ctags command is installed in a system path. On
 unix systems, distributions usually provide the ctags package; on Windows, the
 ctags binary can be found in the zip Windows distribution from the ctags home
-page (http://ctags.sourceforge.net).
+page (https://ctags.sourceforge.net). There is also a new, maintained implementation
```
Comment (elextr) [inline geanyctags/README:14]: Given the last release of ectags was 2009 it seems better to not mention it. Uctags has 15 years of improvement and makes Windows and mac binaries.

File: geanyctags/README [hunk 2]
```diff
@@ -109,9 +111,10 @@ Get the code from::
 Ideas, questions, patches and bug reports
 =========================================
 
-Please direct all questions, bug reports and patches to the plugin author using the
-email address listed below or to the Geany mailing list to get some help from other
-Geany users.
+Please direct all questions, bug reports and patches to the combined geany-plugins
+project at https://github.com/geany/geany-plugins and open the corresponding
+bug report or pull request there. To notify the author of this plugin about
+your post, mention him using his github user name (@techee).
```
Comment (elextr) [inline geanyctags/README:117]: Maybe mention that the github username can usually be found in the MAINTAINERS file.

**Review by elextr — COMMENTED** [review]
Couple of suggestions, otherwise looks good.

File: geanyctags/README (hunk 1 above)
Comment (techee) [reply geanyctags/README]: Good point, I've updated this section. I kept the information about ctags though so people know what the difference between ctags and universal ctags is (I think distributions still package the unmaintained ctags).

File: geanyctags/README (hunk 2 above)
Comment (techee) [reply geanyctags/README:117]: I'll probably skip this level of indirection and mention it directly in the documentation. I normally check geany-plugins only from time to time but I have an email notification enabled when someone mentions me.

**Review by elextr — APPROVED** [review]

**Pull request merged**